src/components/SearchBox/SearchBox.jsx

```diff
@@ -0,0 +1,50 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import './SearchBox.scss';
+
+class SearchBox extends Component {
+  constructor(props) {
+    super();
+
+    this.props = props;
+    this.state = {
+      value: '',
+    };
+
+    this.onInputChange = this.onInputChange.bind(this);
+  }
+
+  onInputChange(event) {
+    const { value } = event.target;
+
+    this.setState({ value });
+  }
+
+  render() {
+    const { value } = this.state;
+
+    return (
+      <div className="searchbox">
+        <input
+          onChange={this.onInputChange}
+          placeholder={this.props.placeholder}
+          value={value}
+          spellCheck={false}
+        />
+        <span className="input-highlight">
```

Comment:
I like this but why do we need this? It would make sense if there was a character limit and work as a visual aid but right now its nothing but a glorified cursor

Reply:
The purpose is aesthetics and design to shape the character of the ui. Anyways its not the final design. The project this is a part of is just a first pass at components. Other projects will cover the next phases

Reply:
Im fine with this for now, but I don't think it serves any additional purpose. It may even be confusing as to what this is and what it's trying to tell a user.

Also there's a bug with this. The color doesn't persist onBlur.
<img width="965" alt="bug" src="https://user-images.githubusercontent.com/6703126/40880834-4ba3d9a6-6686-11e8-999a-a0ceb3b05592.PNG">

Reply:
fixed.